src/yamlProjectConfiguration/migration.ts

```diff
@@ -56,6 +57,12 @@ export async function tryV2Migration(config: unknown): Promise<v2 | undefined> {
         } else {
             const scannedClasses = await scanClassesForDecorators({
                 path: v1Config.workspace?.backend || ".",
+                language: {
+                    name: Language.ts,
+                    runtime: "nodejs20.x",
+                    packageManager: PackageManagerType.npm,
```

Comment:
these are not necessary either.

```suggestion
```

Reply:
I actually get this error if I try to remove those `Property 'packageManager' is missing in type '{ name: Language.ts; }' but required in type '{ packageManager: NonNullable<PackageManagerType | undefined>; }'. `